code/modules/mob/living/simple_animal/bot/cleanbot.dm

```diff
@@ -56,8 +57,8 @@
 		user.transferItemToLoc(W, src)
 		weapon = W
 		weapon_orig_force = weapon.force
-		weapon.force = weapon.force / 2
-		icon_state = "cleanbot[on]"
+		if(!emagged)
+			weapon.force = weapon.force / 2
```

Comment:
what if you attach a knife to an already emagged one

Reply:
then it stays at full force, which is the intended behavior. The other possibility (emagging an already armed bot) is taken care of on line 170